build/IPDK_Container/Ubuntu20.04/scripts/build_p4c.sh

```diff
@@ -0,0 +1,41 @@
+#Copyright (C) 2021 Intel Corporation
```

Comment:
Better to reuse the existing build script instead of adding any new file here unless specific changes are added. From an initial analysis, I see it is the same as the existing build script and didn't see any difference.

Reply:
Removed from Ubuntu20.04 folder.